src/utils/newrelic.js

```diff
@@ -0,0 +1,34 @@
+'use strict';
+let _newrelic;
+
+// NewRelic initialization
+function init() {
+  if (process.env.NEW_RELIC_ENABLED) {
+    process.env.NEW_RELIC_NO_CONFIG_FILE = 'True';
+
+    if (!_newrelic) {
+      // newrelic is cached as a singleton.
+      _newrelic = require('newrelic');
+    }
+  }  
+  return _newrelic;
+}
+
+function crashClose() {
+  if (!_newrelic) {
+    return Promise.resolve();
+  } else {
+    return new Promise(resolve => {
+      _newrelic.addCustomParameter('crash', 'true');
+      _newrelic.agent.harvest(() => {
+        process.exit(-1);
```

Comment:
No need to ``process.exit`` here - just ``_newrelic.agent.harvest(resolve)``
``serverRunner`` will exit after everyone is ready